.github/workflows/cd.yml

```diff
@@ -28,6 +28,11 @@ jobs:
             target: x86_64-unknown-linux-musl
             binary_postfix: ""
             label: x86_64 musl
+          - os: ubuntu-latest
+            artifact_prefix: linux-arm64
+            target: aarch64-unknown-linux-gnu
```

Comment:
We might as well add `aarch64-unknown-linux-musl` as well, since this project also provides musl builds for linux

Reply:
Added!